src/Compilers/CSharp/Test/Semantic/Semantics/ReadOnlyStructsTests.cs

```diff
@@ -817,6 +1305,211 @@ public struct S
                 Diagnostic(ErrorCode.ERR_BadMemberFlag, "S").WithArguments("readonly").WithLocation(5, 21));
         }
 
+        [Fact]
+        public void ReadOnlyStruct_Constructor()
+        {
+            var csharp = @"
+public readonly struct S
+{
+    public readonly int i;
+    public S(int i)
```

Comment:
Is `readonly` prohibitied on constructors? Do we have a test for that?

Reply:
See ReadOnlyStructsTests.ReadOnlyConstructor